### ISSUE TITLE
NTR: Treponemal antibody titer high in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8734]
New term request Treponemal antibody titer high in blood (samant_101126232605): TPHA; Treponema pallidum hemoagglutination assay; TPPA; Treponema pallidum particle agglutination test; Fluorescent treponemal antibody absorption (FTA-ABS); syphilis  UMLS: 0749676

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000926
- label: Anti-Treponema pallidum antibody positivity
- parents: [Positive bloodstream antibacterial antibody positivity (HP:0430061)](https://hpo.jax.org/app/browse/term/HP:0430061)
- pmid: 
- def: The presence of antibodies in the blood circulation that react against a component of Treponema pallidum.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8734
- synonyms: Treponemal antibody titer high in blood(NONE)|Positive Fluorescent treponemal antibody absorption test(NONE)|Positive Treponema pallidum hemoagglutination assay(NONE)
